boards/microbit_v2/openocd.cfg

```diff
@@ -6,6 +6,8 @@ set WORKAREASIZE 0x40000
 
 $_TARGETNAME configure -work-area-phys 0x20000000 -work-area-size $WORKAREASIZE -work-area-backup 0
 
+# comment the following lines if you are using openocd 0.11 or newer
```

Comment:
I think we can use if statements in cfg files. Is there a way to automatically include these lines based on version number?

Alternatively, should they be disabled by default, since new people will install openocd version 0.11.

Reply:
Also, why are we referencing the nrf51 here anyways?

Reply:
I don't know, this seems to work. I used the configuration file provided by the Arduino IDE plugin for the Micro:bit.

> I think we can use if statements in cfg files. Is there a way to automatically include these lines based on version number?
> 
Still looking into this, but I don't seem to find a lot of documentation.
> Alternatively, should they be disabled by default, since new people will install openocd version 0.11.
It was released in March, Ubuntu still provides 0.10. The 0.11 si in rc-2.

Reply:
> Also, why are we referencing the nrf51 here anyways?

I used the configuration file from the Arduino IDE plugin for Micro:bit. This seems to work.

Reply:
Fixed it, I placed a catch around the two lines that show errors. It will not print an error, but work. I suggest we delete them after a while, when 0.11 becomes more popular.

Reply:
Awesome. That's exactly what I was thinking.